frontend/projects/upgrade/src/styles.scss

```diff
@@ -242,6 +242,19 @@ button[mat-button], button[mat-raised-button], button[mat-flat-button], button[m
   width: 805px;
 }
 
+.mat-mdc-checkbox {
+  .mdc-checkbox {
+    transform: scale(0.889); // transform 18px to 16px
+  }
+
+  span {
```

Comment:
can we give these spans a class instead of targeting an html element? I've always followed the school of thought that classes are ideal unless you can't avoid it. Unless angular creates class-less spans dynamically at runtime? but I think these spans are the ones you've added in the template.

Reply:
I like that idea. I will add a class called `checkbox-label` to the existing span elements within the `mat-checkbox`.